frontend/src/app/components/wp-relations/wp-relations-hierarchy/wp-relations-hierarchy.service.ts

```diff
@@ -136,7 +136,11 @@ export class WorkPackageRelationsHierarchyService {
         lockVersion: childWorkPackage.lockVersion
       }).then(wp => {
         if (parentWorkPackage) {
-          this.wpCacheService.require(parentWorkPackage.id!, true);
+          this.wpCacheService.loadWorkPackage(parentWorkPackage.id!, true);
```

Comment:
Is there a reason to use the `loadWorkpackage` method instead of `require`? As far as I can see, the former is a wrapper around the later, with the extend of returning a state which is not used here.